README.md

```diff
@@ -21,6 +21,12 @@ Safety countermeasures:
 
 ## How to run it
 
+You have two options to run it:
+
+1. Manually install and run the [`aws-cloud-unmap` Python package](https://pypi.org/project/aws-cloud-unmap/)
+2. Use the [Docker image available on Docker hub](https://hub.docker.com/u/spreaker/aws-cloud-unmap/)
+
```

Comment:
I think it's easier to include the exact commands to install and run for the first time. That's something I really appreciate when reading a README

Reply:
How does it look, now?

Reply:
Way better! Just after the `pip3`, you should add a running example similar to the docker one.